nixos/modules/virtualisation/digital-ocean-config.nix

```diff
@@ -87,7 +87,6 @@ with lib;
               echo "metadata unavailable, trying again in 1s..."
               sleep 1
             done
-            chmod 600 $RUNTIME_DIRECTORY/v1.json
```

Comment:
I guess, I have now the right setup in nixos-anywhere to test this automatically.

Reply:
any updates?